odysseus/odysseus_tree/sources/tpu_telemetry/poll_data/on_board.py

```diff
@@ -0,0 +1,63 @@
+
+import psutil, sys
+
+# fetch_data() -> List[(str, [str], str)]
+
+def fetch_data():
+    return[fetch_cpu_temperature(), fetch_cpu_usage(), fetch_broker_cpu_usage(), fetch_available_memory()]
+    
+
+
+# CPU Temp
+def fetch_cpu_temperature():
+    temps = psutil.sensors_temperatures(fahrenheit=False)
+    
+    if not temps:
+        sys.exit("can't read any temperature")
```

Comment:
can you make this just print the error and return a None list like the other functions?

Reply:
just fixed them